### ISSUE TITLE
Ringtone not working

### ISSUE THREAD

**Issue by ultr41337h4xor** [SimpleMobileTools/Simple-Dialer#195]
Hey there. I got a weird problem on my new phone - Fairphone 3+ with LineageOS flashed on it. The ringtone I set in Android settings doesn't seem to work for Simple Dialer. Every time I'm getting a call I just get an unfamiliar single beep-like sound on repeat. The ringtone works on other apps (for example, for Signal voice calls). Anyone got any ideas?

Awesome apps btw, thanks for all the hard work!

**Comment by tibbi** [comment]
What app version are you using?

**Comment by ultr41337h4xor** [comment]
> What app version are you using?

F-Droid version 5.7.0

**Comment by ultr41337h4xor** [comment]
@tibbi Any updates on this? It's a really bad bug for me, since I can't even hear incoming calls most of the time, because it's just that short ding sound.

New update didn't change anything. Maybe adding an in-application option of changing the ringtone?

I can take a video of the sound when a call is incoming if you'd like.

**Comment by tibbi** [comment]
I dont know, seems like a device specific rooting issue. Do other Dialer apps work well?

**Comment by ultr41337h4xor** [comment]
@tibbi Sorry for the radio silence. A few days ago my LineageOS version got an update to the latest version 18.1 (Android 11), so I wanted to wait until I upgraded to see if that fixed the problem.

The first call I made, it was the same thing as before. After that I went to system settings and selected the same ringtone that was set before and that I wanted to use. The text changed from "<filename>.m4a" to "<filename>" and now it works.

My guess is it was either the app or the OS having a problem finding the file that should be used as a ringtone and replacing it with a default sound when it couldn't. Hope this helps.
